logging/google/cloud/logging/handlers/middleware/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Yeah this is a Django middleware, and I think it's better to live in a `middleware` folder rather than stay with other files in the `handlers`.